torchmultimodal/modules/layers/transformer.py

```diff
@@ -29,6 +29,284 @@
 )
 
 
+class TransformerEncoderCrossAttentionLayer(nn.Module):
```

Comment:
```suggestion
class TransformerCrossAttentionLayer(nn.Module):
```
encoder might be confusing since it takes in another encoder input to cross attend to